pkg/crc/preflight/preflight.go

```diff
@@ -5,13 +5,13 @@ import (
 	cmdConfig "github.com/code-ready/crc/cmd/crc/cmd/config"
 	"github.com/code-ready/crc/pkg/crc/config"
 	"github.com/code-ready/crc/pkg/crc/output"
-	"os"
+	log "github.com/sirupsen/logrus"
 )
 
 type PreflightCheckFixFuncType func() (bool, error)
 
 func preflightCheckSucceedsOrFails(isConfiguredToSkip bool, check PreflightCheckFixFuncType, message string, isConfiguredToWarn bool) {
-	output.Out("-- %s ... ", message)
+	log.Infof("%s ... ", message)
```

Comment:
So, this is how i am planning us to use the logrus pkg, whenever we need to log to conosle/file
Everything is logged to file by, and all info msgs are logged to console by default which can be changed using the `log-level` global flag